CHANGES.rst

```diff
@@ -54,6 +54,9 @@ Bugs fixed
   get passed to :program:`latexmk`.  Let :option:`-Q <sphinx-build -Q>`
   (silent) apply as well to the PDF build phase.
   Patch by Jean-François B.
+* #12744: Classes injected by a custom interpreted text role now give rise to
+  nested ``\DUrole``'s, rather than a single one with comma separated classes.
```

Comment:
```suggestion
* #12744: LaTeX: Classes injected by a custom interpreted text role now give
  rise to nested ``\DUrole``'s, rather than a single one with comma separated
  classes.
```